prow/config/config.yaml

```diff
@@ -2039,6 +2051,21 @@ tide:
         - do-not-merge/blocked-paths
         - do-not-merge/needs-linked-issue
         - needs-rebase
+    - repos:
+        - pingcap/tiflash
+      includedBranches:
+        - planner_refactory
+      labels:
+        - status/can-merge
+        - cherry-pick-approved
```

Comment:
remove `cherry-pick-approved`, I think we don't need to cherry-pick PR from master to this feature branch.